.magicignore

```diff
@@ -0,0 +1,5 @@
+.git
+.libs
+rock_tmp
```

Comment:
There are no ooc files in these folders, afaik. Does adding them here improve magic's performance?